### DISCUSSION TITLE
https://raw.githubusercontent.com not working for certain network providers in India (including jio the biggest)

### DISCUSSION THREAD

**Discussion by yogeshdofficial** [community/community#58355] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I know I can just my dns but Is there no other fixes and why does that happens?

**Comment by aahouzi** [comment]
If you're experiencing issues accessing https://raw.githubusercontent.com from certain network providers in India, including Jio, it's possible that there could be network restrictions or blocking in place. This type of blocking can sometimes occur due to various reasons, including regional internet restrictions or network configurations.

Here are a few steps you can take to try and resolve the issue:

1. Use a VPN: Consider using a virtual private network (VPN) service to bypass any network restrictions or blocking. A VPN can help route your internet traffic through a different network location, potentially allowing you to access https://raw.githubusercontent.com without any restrictions.

2. Contact your network provider: Reach out to your network provider, such as Jio, and inquire about any restrictions or blocking that might be in place. They should be able to provide you with more information and potential solutions.

3. Use alternative DNS servers: Try changing your DNS (Domain Name System) servers to alternative providers, such as Google DNS (8.8.8.8 and 8.8.4.4) or Cloudflare DNS (1.1.1.1 and 1.0.0.1). Instructions for changing DNS servers can vary depending on your operating system or network setup.

4. Use a different network: If possible, try accessing https://raw.githubusercontent.com from a different network or internet connection. This can help determine if the issue is specific to your current network provider.

4. Report the issue to GitHub: If the problem persists, consider reporting the issue to GitHub support or their community forums. They may be able to provide more insight or investigate any potential issues with accessing their services from certain network providers in India.
